nix/mobile/android/build.nix

```diff
@@ -109,6 +109,10 @@ in stdenv.mkDerivation rec {
     export ANDROID_NDK_ROOT="${androidPkgs.ndk}"
 
     export STATUS_NIX_MAVEN_REPO="${deps.gradle}"
+
+    # Sentry vars relevant for release
+    export SENTRY_CONTEXT_NAME="status-mobile"
```

Comment:
Shouldn't we also export vars for iOS builds?

Reply:
This change is not required, you may revert changes to this file